SYCL/ESIMD/private_memory/pm_access_1.cpp

```diff
@@ -7,7 +7,6 @@
 //===----------------------------------------------------------------------===//
 
 // REQUIRES: gpu
-// Sporadic failure in GPU RT 21.16.19610
-// UNSUPPORTED: cuda || (level_zero && linux) || (opencl && linux)
-// RUN: %clangxx -fsycl -I%S/.. %S/Inputs/pm_common.cpp -o %t.out
+// UNSUPPORTED: cuda
+// RUN: %clangxx -fsycl -Xs "-stateless-stack-mem-size=131072" -I%S/.. %S/Inputs/pm_common.cpp -o %t.out
```

Comment:
Will the options work is CL-style clang is used (aka lang-cl)?
If not mark them UNSUPPORTED: cl_options

Reply:
I believe option should work fine when CL-style clang is used. Jenkins/pre-ci-windows successfully passed, so I think marking unsupported is not needed.